### ISSUE TITLE
Improve the new block inserter "Tips" semantics

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#22863]
**Describe the bug**
In the new block inserter, some "tips" are displayed at the bottom of the inserter:

<img width="372" alt="Screenshot 2020-06-03 at 17 29 36" src="https://user-images.githubusercontent.com/1682452/83656648-130acc00-a5c0-11ea-8dca-09bf917ec2d5.png">

The tips text is just text within a paragraph element. There's nothing to explain to non-sighted users what this text actually _is_.

For example, screen reader users navigating the inserter content will find at some point some text that says: 

"Change a block's type by pressing the block icon on the toolbar."

At the very least, this text should have other text befor it to explain what it is. Preferably a (visually hidden) heading. See also https://github.com/WordPress/gutenberg/issues/22859 for the importance of headings. 

Also: related to the tips content: for non sighted users it doesn't make much sense and I'd say it's also slightly unfair to refer to a "block icon" to press, as they can't see what icon this message refers to

Any instruction on usage needs to avoid references to visual aspect, position, etc. as they're things non-sighted users can't perceive. For this reason, I'd like to recommend also to review all the messages used as tips and refactor them in a way that they can be understood by everyone.

**To reproduce**
- install and activate the Gutenberg plugin 8.2.1
- edit a post
- open the main inserter
- see the Tips at the bottom of the inserter

**Expected behavior**
The tips to be preceeded by a heading, as any relevant content section should be identified by a heading.

**Editor version (please complete the following information):**
- WordPress version: 5.4
- Gutenberg plugin 8.2.1

**Comment by gaambo** [comment]
Hi, first-time contributor here - I just added a PR to add a heading wrapped in a `VisuallyHidden` component. Since in #22859 @afercia mentioned the headlines should be `h2` I chose a `h2` here as well. 

I've added a copy which probably needs review and translation - is there any way I need to add this? 
I hope the `VisuallyHidden` component is the correct one - just seemed naturally to me. 

Happy awaiting any feedback since this is my first contribution during WCEU contributor day 👍

**Comment by afercia** [comment]
Thanks @gaambo! I left a comment on your PR.

**Comment by ZebulanStanphill** [comment]
Fixed by #22898.
